subgraphs/openzeppelin-governor/protocols/unlock-governance/src/unlock-protocol-governor.ts

```diff
@@ -64,7 +64,7 @@ export function handleProposalExecuted(event: ProposalExecuted): void {
 
 // ProposalQueued(proposalId, eta)
 export function handleProposalQueued(event: ProposalQueued): void {
-  _handleProposalQueued(event.params.proposalId, event.params.eta);
+  _handleProposalQueued(event.params.proposalId, event.params.eta, event);
```

Comment:
Feels weird to pass 3 args -- 2 of which are fields from event and then pass the entire event as well

Should either:
1) pass only the event fields we care about
or
2) pass the entire event and let the fn take what it needs (probably more forward compatible)

Reply:
i agree about the weirdness, but this method allows us to have typed args for the generalised handler. `event.params` are unique to the contract while the `_handleProposalQueued` function uses a generic `ethereum.event`.

The alternative is to fully list out each arg of the event args like txnHash, block number, and timestamp but we're already at 10+ args for some functions.

Reply:
Maybe its worth passingin event.transaction and event.block then? That way the handlers get only what they need

The handlers then receive param specific items (proposalId + eta), and transaction + block which feels cleaner from a data perspective

Not too heavily opinionated on it so if you disagree lets roll with what you have

Reply:
I'm okay to make the change, but could we do it in a separate future PR?

Reply:
Lets keep it as is then